jenkins_job_wrecker/modules/properties.py

```diff
@@ -250,3 +251,22 @@ def authorizationmatrixproperty(top, parent):
         else:
             raise NotImplementedError('cannot handle XML {}'.format(child.tag))
     parent.append({'authorization': authorization})
+
+
+def buildblockerproperty(top, parent):
+    build_blocker_property = {}
+    for child in top:
+        if child.tag == "useBuildBlocker":
+            build_blocker_property["use-build-blocker"] = get_bool(child.text)
+        elif child.tag == "blockLevel":
+            build_blocker_property["block-level"] = child.text
+        elif child.tag == "scanQueueFor":
+            build_blocker_property["queue-scanning"] = child.text
+        elif child.tag == "blockingJobs":
+            blocking_jobs = []
+            if child.text is not None:
+                blocking_jobs = child.text.replace("\n", " ")
```

Comment:
I think it's possible for a job to have a space in the name. In this case it would be better to just use something like `[name.strip() for name in child.text.split('\n')]` (not tested).

Reply:
You are right, I thought space was %20 in the jobs. I will handle this. Thanks for noticing it.